specificatie/domain.yaml

```diff
@@ -203,6 +203,15 @@ components:
                         De identificaties van de stukdelen (paragrafen in een akte met een rechtsfeit) waarin dit beslag is vermeld
           items:
             type: "string"
+    BijbehorendeAppartementsrechten:
```

Comment:
BijbehorendeAppartementsrechten => BijbehorendeAppartementsrecht? Dit is een object en niet een array van objecten

Reply:
een vervallen appartementsrecht wordt niet geleverd, dus we kunnen geen appartementsrecht krijgen met indicatieVervallen: true.

Reply:
> 
> 
> een vervallen appartementsrecht wordt niet geleverd, dus we kunnen geen appartementsrecht krijgen met indicatieVervallen: true.

Wordt wel verwarrend. Uit de filiatie feature lees ik dat vervallen perceel wel wordt geleverd. Omdat een perceel en appartementsrecht als KOZ is gemoduleerd en een vervallen perceel ook met /kadastraalonroerendezaken/{identificatie} wordt bevraagd, moet volgens mij indicatieVervallen ook aan de KadastraalOnroerendeZaak als property worden toegevoegd. Dit staat volgens mij ook in de filiatie feature (regel 98). Er wordt hierdoor een property toegevoegd die alleen maar bij perceel type kan voorkomen en bij appartementsrecht type niet.

Reply:
> Wordt wel verwarrend

het is eigenlijk nog "erger". Wanneer een perceel wordt gesplitst of samengevoegd, vervalt de oude (en blijft wel raadpleegbaar). Wanneer een appartementsrecht wordt gesplitst, wordt de oude "sluimerend". Wanneer appartementsrechten worden samengevoegd, vervallen de oude (en zijn niet meer raadpleegbaar). Dat is misschien niet erg, want deze is ook niet meer zichtbaar (wordt niet opgenomen in isOvergegaanIn of isOntstaanUit).

Maar een gebruiker moet dus ergens begrijpen dat vervallen en sluimerend eigenlijk hetzelfde is: de onroerende zaak is overgegaan in andere onroerende zaak/zaken.

@CathyDingemanse @kad-pothot kunnen we een sluimerend appartementsrecht niet gewoon "vervallen" noemen? Of is dat te verwarrend voor gebruikers die wel kennis hebben van filiatie in BRK?

Reply:
Kan je niet vervallen noemen, omdat er ook nog een ander actief recht op kan staan, inclusief eigenaar. Daarom dekt sluimerend beter de lading.

Reply:
Conclusie overleg dd 30-06-2021: We gaan indicatieSluimerend niet in versie 1.3.0 leveren